cmd/ignite/run/create_test.go

```diff
@@ -120,53 +251,28 @@ func TestConstructVMFromCLI(t *testing.T) {
 					PublicKey: "some-pub-key",
 				},
 			},
-			args: []string{testImage},
 			wantSSH: api.SSH{
 				Generate:  true,
 				PublicKey: "some-pub-key",
 			},
 		},
-		{
-			name: "with no VM name and --require-name flag set",
-			createFlag: &CreateFlags{
-				VM:          &api.VM{},
-				RequireName: true,
-			},
-			args: []string{testImage},
-			err:  true,
-		},
-		{
-			name: "with sandbox image",
```

Comment:
This is already tested in e2e so it's fine to exclude it here (tricky to implement).